internal/backend/dashboardsvc/connections.go

```diff
@@ -12,10 +13,21 @@ import (
 )
 
 func (s Svc) initConnections() {
+	// These paths should correspond to the ones enriched in the connection service.
+
 	s.Muxes.Auth.HandleFunc("GET /connections", s.connections)
 	s.Muxes.Auth.HandleFunc("GET /connections/{cid}", s.connection)
 	s.Muxes.Auth.HandleFunc("GET /connections/{id}/init", s.initConnection)
+	s.Muxes.Auth.HandleFunc("GET /connections/{id}/postinit", s.postInitConnection)
 	s.Muxes.Auth.HandleFunc("DELETE /connections/{id}/vars", s.rmAllConnectionVars)
+
+	// GET for browser, POST for automated.
+	s.Muxes.Auth.HandleFunc("GET /connections/{id}/test", s.testConnection)
```

Comment:
this endpoints are also exposed on GRPC, why do you re implement them in http instead of using connect http endpoints ?

Reply:
yeah i worry people might treat this as a legit API, but it's very easy to use for the ui. let me see if i can get rid of it.